### ISSUE TITLE
Tuplets with a space and a note are not valid

### ISSUE THREAD

**Issue by wergo** [music-encoding/music-encoding#577]
I came across a Schematron error that occurs when you have only a space and a note within a tuplet, as sometimes required for secondary voices in CMN piano repertoire.

<img width="838" alt="tuplet-with-one-element" src="https://user-images.githubusercontent.com/45632600/53271052-8fd8a600-36ed-11e9-9ab7-f65092c57a06.png">

Please see my example in which I made the tuplet brackets and numbers visible for better explanation. The second layer contains tuplets with a `<space dur="8">` and a `<note dur="4">` enclosed by a `<tuplet num="3" numbase="2">`. 

The Schematron error message is "A tuplet without a copyof attribute must have at least 2 note, rest, or chord descendants."

Of course, as in bar 2 and 3, you could code the three 3:2 tuplets as one 9:6 tuplet with spaces and notes, as I did in bar 3. This works without error message, but the bar 1 example does not have such a work-around.

I suggest to add the element space into the list of accepted elements of tuplets so that the error message would read  "A tuplet without a copyof attribute must have at least 2 note, rest, space, or chord descendants." and will not be thrown in my example.

Here is the sample MEI code: 

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/4.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="4.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title>Tuplet with only one element</title>
            </titleStmt>
            <pubStmt>
                <respStmt>
                    <persName role="encoder" auth.uri="http://d-nb.info/gnd" auth="GND" codedval="1138881465">Werner Goebl</persName>
                </respStmt>
            </pubStmt>
        </fileDesc>
    </meiHead>
    <music>
        <body>
            <mdiv xml:id="mdiv-0000001665096923">
                <score xml:id="score-0000001424580804">
                    <scoreDef xml:id="scoredef-0000001268321234" key.sig="3s" meter.count="3" meter.unit="4">
                        <staffGrp xml:id="staffgrp-0000000634392425">
                            <staffDef xml:id="staffdef-0000000409533897" clef.shape="F" clef.line="4" n="2" lines="5"/>
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000002000015636">
                        <measure xml:id="measure-0000000497854352" n="30" right="dbl">
                            <staff xml:id="staff-0000000815747560" n="2">
                                <layer xml:id="layer-0000000733638472" n="1">
                                    <tuplet xml:id="tuplet-0000001558181477" num="3" numbase="2" num.visible="false" bracket.visible="false">
                                        <beam xml:id="beam-0000002076440069">
                                            <note xml:id="note-0000001940492421" dur="8" oct="3" pname="e" stem.dir="up"/>
                                            <note xml:id="note-0000002118975933" dur="8" oct="2" pname="a" stem.dir="up"/>
                                            <note xml:id="note-0000001907187730" dur="8" oct="3" pname="e" stem.dir="up"/>
                                        </beam>
                                    </tuplet>
                                    <beam xml:id="beam-0000000763262988">
                                        <tuplet xml:id="tuplet-0000001241215785" num="3" numbase="2" num.visible="false" bracket.visible="false">
                                            <note xml:id="note-0000000457551537" dur="8" oct="3" pname="e" stem.dir="up">
                                                <accid xml:id="accid-0000002077226099" accid="s"/>
                                            </note>
                                            <note xml:id="note-0000000297396614" dur="8" oct="3" pname="d" stem.dir="up">
                                                <accid xml:id="accid-0000001150444929" accid="x" accid.ges="ss"/>
                                            </note>
                                            <note xml:id="note-0000001732647762" dur="8" oct="3" pname="e" stem.dir="up" accid.ges="s"/>
                                        </tuplet>
                                    </beam>
                                    <beam xml:id="beam-0000000525461600">
                                        <tuplet xml:id="tuplet-0000000980354736" num="3" numbase="2" num.visible="false" bracket.visible="false">
                                            <note xml:id="note-0000001327508168" dur="8" oct="3" pname="f" stem.dir="up" accid.ges="s"/>
                                            <note xml:id="note-0000000345115096" dur="8" oct="3" pname="e" stem.dir="up" accid.ges="s"/>
                                            <note xml:id="note-0000000821748532" dur="8" oct="3" pname="f" stem.dir="up" accid.ges="s"/>
                                        </tuplet>
                                    </beam>
                                </layer>
                                <layer xml:id="layer-0000000663945820" n="2">
                                    <tuplet xml:id="tuplet-0000000612366928" num="3" numbase="2" num.visible="true" bracket.visible="true">
                                        <space xml:id="space-0000001309322472" dur="8"/>
                                        <note xml:id="note-0000000517856095" dur="4" oct="2" pname="a" stem.dir="down"/>
                                    </tuplet>
                                    <note xml:id="note-0000000671641340" dur="2" oct="2" pname="a" stem.dir="down"/>
                                </layer>
                            </staff>
                            <slur xml:id="slur-0000000021972758" startid="#note-0000001940492421" endid="#note-0000000821748532" curvedir="above"/>
                            <tie xml:id="tie-0000002003651021" startid="#note-0000000517856095" endid="#note-0000000671641340" curvedir="below"/>
                        </measure>
                        <measure xml:id="measure-0000001620164877" n="37">
                            <staff xml:id="staff-0000001534514813" n="2">
                                <layer xml:id="layer-0000001459345268" n="1">
                                    <tuplet xml:id="tuplet-0000000805186889" num="3" numbase="2" num.visible="false" bracket.visible="false">
                                        <rest xml:id="rest-0000001481583676" dur="8"/>
                                        <beam xml:id="beam-0000000903955567">
                                            <note xml:id="note-0000001481895691" dur="8" oct="3" pname="b" stem.dir="up"/>
                                            <note xml:id="note-0000001853024378" dur="8" oct="4" pname="d" stem.dir="up"/>
                                        </beam>
                                    </tuplet>
                                    <tuplet xml:id="tuplet-0000000373535540" num="3" numbase="2" num.visible="false" bracket.visible="false">
                                        <beam xml:id="beam-0000001555772874">
                                            <note xml:id="note-0000000917120599" dur="8" oct="4" pname="d" stem.dir="up"/>
                                            <note xml:id="note-0000000113807446" dur="8" oct="3" pname="a" stem.dir="up">
                                                <accid xml:id="accid-0000001501299092" accid="n"/>
                                            </note>
                                            <note xml:id="note-0000001548470641" dur="8" oct="4" pname="d" stem.dir="up"/>
                                        </beam>
                                    </tuplet>
                                    <tuplet xml:id="tuplet-0000000261500868" num="3" numbase="2" num.visible="false" bracket.visible="false">
                                        <beam xml:id="beam-0000001662663617">
                                            <note xml:id="note-0000001293546714" dur="8" oct="4" pname="d" stem.dir="up"/>
                                            <note xml:id="note-0000001330196155" dur="8" oct="3" pname="g" stem.dir="up">
                                                <accid xml:id="accid-0000001302011815" accid="n"/>
                                            </note>
                                            <note xml:id="note-0000000054211775" dur="8" oct="4" pname="d" stem.dir="up"/>
                                        </beam>
                                    </tuplet>
                                </layer>
                                <layer xml:id="layer-0000002093399263" n="2">
                                    <tuplet xml:id="tuplet-0000001536824440" num="3" numbase="2" num.visible="true" bracket.visible="true">
                                        <space xml:id="space-0000001622540611" dur="8"/>
                                        <note xml:id="note-0000001292699471" dur="4" oct="3" pname="b" stem.dir="down"/>
                                    </tuplet>
                                    <tuplet xml:id="tuplet-0000000307952398" num="3" numbase="2" num.visible="true" bracket.visible="true">
                                        <space xml:id="space-0000000320363916" dur="8"/>
                                        <note xml:id="note-0000000614833183" dur="4" oct="3" pname="a" stem.dir="down">
                                            <accid xml:id="accid-0000001957480964" accid="n"/>
                                        </note>
                                    </tuplet>
                                    <tuplet xml:id="tuplet-0000002080573555" num="3" numbase="2" num.visible="true" bracket.visible="true">
                                        <space xml:id="space-0000000723514784" dur="8"/>
                                        <note xml:id="note-0000001060565374" dur="4" oct="3" pname="g" stem.dir="down" accid="n"/>
                                    </tuplet>
                                </layer>
                            </staff>
                            <tie xml:id="tie-0000000972872252" startid="#note-0000001853024378" endid="#note-0000000917120599"/>
                            <slur xml:id="slur-0000000123451106" startid="#note-0000001481895691" endid="#note-0000001035990273" curvedir="above"/>
                            <tie xml:id="tie-0000001939228941" startid="#note-0000001548470641" endid="#note-0000001293546714"/>
                            <tie xml:id="tie-0000000604236097" startid="#note-0000000054211775" endid="#note-0000000173302777"/>
                        </measure>
                        <measure xml:id="measure-0000000980665267" n="38">
                            <staff xml:id="staff-0000001629138926" n="2">
                                <layer xml:id="layer-0000000521430032" n="1">
                                    <tuplet xml:id="tuplet-0000001941268064" num="3" numbase="2" num.visible="false" bracket.visible="false">
                                        <beam xml:id="beam-0000000711947707">
                                            <note xml:id="note-0000000173302777" dur="8" oct="4" pname="d" stem.dir="up"/>
                                            <note xml:id="note-0000002073714112" dur="8" oct="3" pname="f" stem.dir="up" accid.ges="s"/>
                                            <note xml:id="note-0000001146219639" dur="8" oct="4" pname="d" stem.dir="up"/>
                                        </beam>
                                    </tuplet>
                                    <tuplet xml:id="tuplet-0000000086583299" num="3" numbase="2" num.visible="false" bracket.visible="false">
                                        <beam xml:id="beam-0000001375674626">
                                            <note xml:id="note-0000001359077274" dur="8" oct="4" pname="d" stem.dir="up"/>
                                            <note xml:id="note-0000001154495580" dur="8" oct="3" pname="e" stem.dir="up"/>
                                            <note xml:id="note-0000001092462415" dur="8" oct="4" pname="d" stem.dir="up"/>
                                        </beam>
                                    </tuplet>
                                    <tuplet xml:id="tuplet-0000001500321752" num="3" numbase="2" num.visible="false" bracket.visible="false">
                                        <beam xml:id="beam-0000001634578660">
                                            <note xml:id="note-0000000154702790" dur="8" oct="4" pname="d" stem.dir="up"/>
                                            <note xml:id="note-0000001752726196" dur="8" oct="3" pname="a" stem.dir="up"/>
                                            <note xml:id="note-0000001035990273" dur="8" oct="4" pname="c" stem.dir="up" accid.ges="s"/>
                                        </beam>
                                    </tuplet>
                                </layer>
                                <layer xml:id="layer-0000000103626734" n="2">
                                    <tuplet xml:id="tuplet-0000000045280621" num="9" numbase="6" num.visible="true" bracket.visible="true">
                                        <space xml:id="space-0000000822186109" dur="8"/>
                                        <note xml:id="note-0000001572149165" dur="4" oct="3" pname="f" stem.dir="down" accid.ges="s"/>
                                        <space xml:id="space-0000000769824473" dur="8"/>
                                        <note xml:id="note-0000001998428183" dur="4" oct="3" pname="e" stem.dir="down"/>
                                        <space xml:id="space-0000002050388733" dur="8"/>
                                        <note xml:id="note-0000000213352122" dur="4" oct="3" pname="a" stem.dir="down"/>
                                    </tuplet>
                                    <clef xml:id="clef-0000000391356635" shape="G" line="2"/>
                                </layer>
                            </staff>
                            <tie xml:id="tie-0000001585159083" startid="#note-0000001146219639" endid="#note-0000001359077274"/>
                            <tie xml:id="tie-0000000030627055" startid="#note-0000001092462415" endid="#note-0000000154702790"/>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by bwbohl** [comment]
Hi @wergo thanks for spotting this! It pretty obviously seems to be an oversight. Although we will not fix it in v4.0.1, as we want the critical bugfix asap, we will fix this in a soon to come bugfix release.

**Comment by bwbohl** [comment]
fixed with 35c27fa
